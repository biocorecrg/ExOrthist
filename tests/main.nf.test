nextflow_pipeline {

    name "Test pipeline with default settings"
    script "../main.nf"

    test("Params: default") {

        when {
            params {
                output = "$output"
            }
        }

        then {
            assertAll(
                { assert workflow.success }
            )
        }
    }
}
