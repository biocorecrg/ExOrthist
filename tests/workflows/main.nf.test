nextflow_workflow {

    name "Test Workflow MAIN"
    script "workflows/main.nf"
    workflow "MAIN"

    test("Should run without failures") {

        when {
            params {
            }
            workflow {
            }
        }

        then {
            assert workflow.success
            assert snapshot(workflow.out).match()
        }

    }

}
