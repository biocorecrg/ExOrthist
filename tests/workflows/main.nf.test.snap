{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    "EX_clusters.tab:md5,02479360ed2b8d659612fc60477ad355"
                ],
                "exon_cluster_for_reclustering": [
                    "EX_clusters.tab:md5,02479360ed2b8d659612fc60477ad355"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.1",
            "nextflow": "24.10.0"
        },
        "timestamp": "2024-11-02T16:22:03.038334081"
    }
}